client/src/styles/Profile.css

```diff
@@ -275,3 +275,37 @@
   display: block;
   margin: auto;
 }
+
+@media (min-width: 768px) {
+  .profile-section-wrapper {
+    background-image: none;
+    min-height: auto;
+    padding-bottom: 0;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .profile-article {
+    width: 100%;
+    margin-top: 5rem;
+    border: none;
+    border-radius: 0;
```

Comment:
Do we need this, since it is set to 0?

Reply:
yes it is to removed the border-radius that has been applied in mobile-size version